### ISSUE TITLE
[Feature Request] (UI) Change bottom navigation and category color

### ISSUE THREAD

**Issue by padamupreti** [inorichi/tachiyomi#4119]
## Navigation and category tabs color change option
With v0.10.6, bottom navigation active icon color and active category text color have changed to blue. White matched the contrast better in dark theme as it was with previous version.

## An option to choose between white or blue color
This option in the library settings, where theme settings for app are also present. Users can choose their preference, dark theme with white icons + categories or with blue icons + categories.

In short, having an option to go back to this:

![inked](https://user-images.githubusercontent.com/59172926/102010072-b56fed00-3d63-11eb-9bc5-805f68b81350.jpg)

**Comment by arkon** [comment]
Having options for every minor stylistic choice like this is not maintainable.
